helios-system-tests/src/main/java/com/spotify/helios/system/MultiplePortJobTest.java

```diff
@@ -92,14 +92,22 @@ public void test() throws Exception {
 
       final JobId jobId2 = createJob(testJobName + 2, testJobVersion, BUSYBOX, IDLE_COMMAND,
                                      EMPTY_ENV, ports2);
-
       assertNotNull(jobId2);
       deployJob(jobId2, testHost());
       final TaskStatus firstTaskStatus2 = awaitJobState(client, testHost(), jobId2, RUNNING,
                                                         LONG_WAIT_SECONDS, SECONDS);
 
-      assertEquals(expectedMapping1, firstTaskStatus1.getPorts());
-      assertEquals(expectedMapping2, firstTaskStatus2.getPorts());
+      // Verify we allocated dynamic ports from within the specified range.
+      assertTrue(portRange.contains(firstTaskStatus1.getPorts().get("foo").getExternalPort()));
+      assertTrue(portRange.contains(firstTaskStatus2.getPorts().get("foo").getExternalPort()));
```

Comment:
These patterns are handy to know, and I'd prefer to use them, but I don't think I can due to the fact that `portRange` is a Guava `Range` and not a `Collection`. I could probably do a song and dance to convert it into some kind of `Collection`, but I think that would be even messier.